packages/sanity/src/core/form/inputs/arrays/ArrayOfPrimitivesInput/ArrayOfPrimitivesInput.tsx

```diff
@@ -154,13 +176,20 @@ export class ArrayOfPrimitivesInput extends React.PureComponent<ArrayOfPrimitive
               <Card padding={1} border>
                 <List
                   onItemMove={this.handleSortEnd}
+                  onItemMoveStart={this.handleItemMoveStart}
+                  onItemMoveEnd={this.handleItemMoveEnd}
                   items={membersWithSortIds.map((m) => m.id)}
                   sortable={isSortable}
                   gap={1}
                 >
                   {membersWithSortIds.map(({member, id}, index) => {
                     return (
-                      <Item key={member.key} id={id} sortable={isSortable} disableTransition>
+                      <Item
+                        key={member.key}
+                        id={id}
+                        sortable={isSortable}
+                        disableTransition={(this.state as any)?.disableTransition}
```

Comment:
with the typing suggestion above you can simplify this (state is never nullish)
```suggestion
                        disableTransition={this.state.disableTransition}
```